internal/configs/version1/__snapshots__/template_test.snap

```diff
@@ -34,7 +34,7 @@ http {
         default $upstream_trailer_grpc_status;
         '' $sent_http_grpc_status;
     }
-    access_log  /dev/stdout  main;
+    access_log ;
```

Comment:
@hafe this seems a bit odd as only allowed configs as per this PR are:
`access_log off;`
`access_log /dev//stdout main`
`access_log syslog:xxxx`

Reply:
I don't know how it works. I just ran "make test-update-snaps" and ended up with this.

Reply:
@hafe can you please add 
`AccessLog:                      "/dev/stdout main",`
after https://github.com/hafe/kubernetes-ingress/blob/feat/access-log-5625/internal/configs/version1/template_test.go#L2041
and then run `make test-update-snaps`. it should generate the correct config